### ISSUE TITLE
SAP HANA Studio, Setup a new project

### ISSUE THREAD

**Issue by lnryan** [SAPDocuments/Tutorials#1415]
This tutorial makes no sense. It misses vital information for a "beginner" that I was only able to advise my colleague on due to reading the full documentation. Namely: it doesn't say that you need to switch to the developer view when in the previous tutorial you were in the Hana Admin View. It doesn't explain what goes into the xsapp file and most significantly it provides instructions that don't match the options available in eclipse - one has to guess that "XS Application Description File" is the xsapp file. (note the space there).

Basically, this "Beginner" tutorial appears to make assumptions at every step that the user has a non-beginner familiarity with the architecture of an XS project.

**Comment by ccmehil** [comment]
Thank you for the feedback, it's unfortunately the result of us creating a tutorial "on the fly" while at an event and then not taking the time to look through it again. I appreciate you bringing it to our attention and we have made the first set up changes and will look to address more as soon as we can.
